CMakeLists.txt

```diff
@@ -80,6 +93,23 @@ if(BUILD_EXAMPLES)
     add_subdirectory(examples)
 endif()
 
+# ==================================================================
+# CMake Summary
+# ==================================================================
+
+message("")
```

Comment:
Empty lines in CMake output is not common, in my experience.

Reply:
This is to distinguish the summaries from `Kokkos` and `KokkosFFT` as shown.

```
-- Setting default Kokkos CXX standard to 17
-- Kokkos version: 4.2.0
-- The project name is: Kokkos
-- Using internal gtest for testing
-- Configured git information in /home/i18048/jh220031a/github/benchmark/github/kokkos-fft/build/generated/Kokkos_Version_Info.cpp
-- Using -std=gnu++17 for C++17 extensions as feature
-- Built-in Execution Spaces:
--     Device Parallel: NoTypeDefined
--     Host Parallel: Kokkos::OpenMP
--       Host Serial: NONE
-- 
-- Architectures:
--  SKX
-- Using internal desul_atomics copy
-- Kokkos Devices: OPENMP, Kokkos Backends: OPENMP

-- KokkosFFT version: 0.0.0
-- KokkosFFT TPLs
   FFTW_OPENMP
```